contracts/libraries/limit/pool/MintLimitCall.sol

```diff
@@ -41,10 +41,11 @@ library MintLimitCall {
         LimitPoolStructs.MintLimitCache memory cache
     ) external {
         if (params.positionId > 0) {
-            if (PositionTokens.balanceOf(cache.constants, msg.sender, params.positionId) == 0)
-                // check for balance held
-                require(false, 'PositionNotFound()');
             cache.position = positions[params.positionId];
+            if (cache.position.liquidity == 0) {
+                // position doesn't exist
+                require(false, 'PositionNotFound()');
+            }
         }
```

Comment:
Instead of reverting when `msg.sender` does not hold the ERC-1155, we now revert if attempting to add liquidity to a position with zero liquidity.

As long as the position exists, any address will be able to add liquidity to it.